### PR TITLE
fix(doc): Minor spelling in `README.md`

### PR DESCRIPTION
**Pull request by sectoramen**
### Context 

Minor spelling fixes

### Description

Fixed a few spelling mistakes

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by sectoramen** [comment]
Fixed a few minor spelling I noticed while going through the ReadMe

File: README.md [hunk 1]
```diff
@@ -239,15 +239,15 @@ Prowler has two parameters related to regions: `-r` that is used query AWS servi
 
 ### Save your reports
 
-1. If you want to save your report for later analysis thare are different ways, natively (supported text, mono, csv, json, json-asff, junit-xml and html, see note below for more info):
+1. If you want to save your report for later analysis there are different ways, natively (supported text, mono, csv, json, json-asff, junit-xml and html, see note below for more info):
 
     ```sh
     ./prowler -M csv
-    ```
+    ```eachg
```
Comment (jfagoagas) [inline README.md]: Is this a typo?

File: README.md [hunk 2]
```diff
@@ -239,15 +239,15 @@ Prowler has two parameters related to regions: `-r` that is used query AWS servi
...
 
     ```sh
     ./prowler -M csv
-    ```
+    ```eachg 
 
     or with multiple formats at the same time:
 
-    ```sh
+    ```eachg sh
```
Comment (jfagoagas) [inline README.md]: Also this.

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Thank you for this review @sectoramen 

Please check the above comments in order to merge!

**Comment by sectoramen** [comment]
Fixed typos

**Review by jfagoagas — APPROVED** [review]
Thanks @sectoramen

**Pull request merged**